### ISSUE TITLE
start_watering service does not work

### ISSUE THREAD

**Issue by calvinbui** [sh00t2kill/linktap_local_http_component#41]
Error is `Failed to call service linktap.start_watering. extra keys not allowed @ data['seconds']. Got None`

```yaml
service: linktap.start_watering
data:
  seconds: 10
target:
  entity_id: valve.linktap_g2s_plants
```

**Comment by sh00t2kill** [comment]
I missed a commit, which is why it didnt fail when i tested.
Will push out a release to fix ASAP.

**Comment by sh00t2kill** [comment]
Fixed in v0.5.3
